src/net_processing.cpp

```diff
@@ -258,7 +268,7 @@ void static ProcessGetData(CNode *pfrom, const Consensus::Params &consensusParam
             // priority messages and we don't want to sit here processing a large number of messages
             // while we hold the cs_main lock, but rather allow these messages to be sent first and
             // process the return message before potentially reading from the queue again.
-            if (inv.type == MSG_BLOCK || inv.type == MSG_FILTERED_BLOCK || inv.type == MSG_THINBLOCK)
```

Comment:
don't we also want high priority handling of graphene and thin blocks?

Reply:
This section of code above only refers to getdata requests.  graphene and thinblocks are not and can not be requested by a getdata...

Reply:
or i should say, only the legacy THINBLOCK requests can be still requested by getdata, but we've had to move away from that for compact blocks.  Perhaps it would be more clear if I swapped the MSG_THINBLOCK for MSG_COMPACT_BLOCK...I'll do that.

Reply:
oh nvm...it's already there on the next line.